frontend/src/app/main/data/dashboard.cljs

```diff
@@ -87,7 +87,7 @@
               ;; to currently active team
               (->> stream
                    (rx/filter (ptk/type? ::du/teams-fetched))
-                   (rx/observe-on :async)
+                   #_(rx/observe-on :async)
```

Comment:
why this code is commented?

Reply:
ups, sorry, this rx/observe-on :async is one of the culprits for the race condition on team fetching

Reply:
Also, this async probably has a reason for being there, you probably fixed one issue and reopened other one that is not evident right now. The problem here is that there are no comment on code for proper explanation of the rationale for that async and that was our fault in the past (probably my fault not documenting it...)